### PR TITLE
27th March 2020 Meeting Minutes

### PR DESCRIPTION
**Pull request by lxcid**
I'll gonna try to get the ball rolling by filing a brief meeting minutes and bring the discussion online. I suggest we use this PR to finalise the todos?

### CODE REVIEW

**Comment by lxcid** [comment]
A few more points:

- sample json data

Want to ask Sheldon, do you think next.js can be use with golang backend?

File: docs/meetings/20200327-minutes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# 27th March 2020 Meeting Minutes
+
+## Attendees
+
+- @subhransu
+- Kajal
```
Comment (subhransu) [inline docs/meetings/20200327-minutes.md]: Kajal is @kajalsinha and Sheldon is @SheldonCheng

**Comment by subhransu** [comment]
I think To-Dos can be added as issues?

**Comment by lxcid** [comment]
Yeah, I was thinking we can file them as separate issues once we finalise them :)

File: docs/meetings/20200327-minutes.md (hunk 1 above)
Comment (lxcid) [reply docs/meetings/20200327-minutes.md]: Noted and updated!

**Comment by subhransu** [comment]
This is what I can think of. 

* Create the base (@kajalsinha is looking into it now) 
* Create contribution guidelines (I can work on it) 
* Find out whether you want to use React / Next or both (@lxcid and @sheldoncheng to discuss)
* Come up with 1st draft of the UI (@sheldon and Yao Long can work together on this) 

Anything else we are missing at the moment?

**Comment by lxcid** [comment]
I think a sample JSON dataset to work with for now?

**Comment by subhransu** [comment]
> I think a sample JSON dataset to work with for now?

Do you have a structure in mind? Or should we wait for the first draft of the UI?
On top of my head I can think of two things:

* People who are looking for jobs 
* Companies/hiring team who are looking for talents. 

### For talent list

- Name
- Skills
- LinkedIn Profile
- Link to CV
- City (this can help us to filter later) 
- Timestamp  

### Job List

- Job Title
- Job Description
- Created by (Person who posted the job)
- City
- Timestamp

**Comment by lxcid** [comment]
It sounds to be like a Want to Buy and Want to Sell setup. I provided a draft structure based on your comment. Also updated the Action Items based on your previous comment.

**Comment by lxcid** [comment]
Initial JSON structures: https://github.com/jobsgowhere/jobsgowhere/pull/1/commits/8061d41ed5acd2f531e381a74bb370ff5d4c2ba8

Latest meeting minutes: https://github.com/lxcid/jobsgowhere/blob/lxcid/20200327-meeting-minutes/docs/meetings/20200327-minutes.md

**Comment by subhransu** [comment]
The structure looks good to me. if we need further changes we can modify incrementally.

**Review by subhransu — APPROVED** [review]

**Comment by subhransu** [comment]
@lxcid I have approved the PR. Go ahead and merge it if no further comments.

**Comment by lxcid** [comment]
I couldn't merge though, have no write access 😅

I update the mock data with more realistic sample data from https://github.com/SingaporeJS/jobs

**Comment by lxcid** [comment]
Ok, I'm done. Feel free to review again and merge if approve. :)

**Pull request merged**